sitemap_generate/generator.py

```diff
@@ -81,21 +83,40 @@ class SitemapGenerator:
     def __init__(self,
                  media_path: str = 'sitemaps',
                  storage: Storage = default_storage,
-                 index_url_name: str = 'sitemap-index',
+                 index_url_name: Optional[str] = None,
+                 sitemaps_view_name: Optional[str] = None,
                  sitemaps: Optional[Dict[str, Type[Sitemap]]] = None):
         """
 
         :param media_path: relative path on file storage
         :param storage: file storage implementation used for sitemaps
         :param index_url_name: name of view serving sitemap index xml file
+        :param sitemaps_view_name: name of view serving indexed sitemaps
         :param sitemaps: mapping: sitemap name -> sitemap implementation
         """
         cls = self.__class__
         self.logger = getLogger(f'{cls.__module__}.{cls.__name__}')
         self.sitemap_root = media_path
         self.storage = storage
-        self.index_url_name = index_url_name
-        self.sitemaps = sitemaps or {}
+        self.index_url_name = (
+            index_url_name
+            if index_url_name is not None
+            else getattr(settings, 'SITEMAP_INDEX_NAME', 'sitemap-index')
```

Comment:
It may be hard to find default values if they are specified in-place. This should be done as a separate app settings module.